### ISSUE TITLE
add possibility to run a local instance of the explorer

### ISSUE THREAD

**Issue by marc0olo** [aeternity/aeproject#423]
as discussed recently

**Comment by marc0olo** [comment]
still think we cannot do this as long we rely on middleware and can't run the node with devmode + middleware activated 🤔 

but assuming it's possible, should we directly aim to integrate with new explorer? @thepiwo

**Comment by marc0olo** [comment]
is there a plan for supporting devmode along with middleware? @jyeshe @sborrazas 

would be awesome to spawn mdw (incl. devmode) along with an instance of aeScan (cc @michele-franchi)

**Comment by sborrazas** [comment]
> is there a plan for supporting devmode along with middleware? @jyeshe @sborrazas
> 
> would be awesome to spawn mdw (incl. devmode) along with an instance of aeScan (cc @michele-franchi)

We currently support adding tests and contracts execution together with devmode for better middleware testing, but this requires special steps to be run when building the middleware image. We can improve the workflow to be able to do this and add better instructions on how to use the middleware together with devmode

**Comment by marc0olo** [comment]
> We can improve the workflow to be able to do this and add better instructions on how to use the middleware together with devmode

this would require a special build, right? I guess we would need a published bundle image similar to what's done with the node right now. I don't think we will see it adopted if we do not have a image for it.

**Comment by sborrazas** [comment]
That's right, we would need to publish a new image that contains both mdw + devmode
